library/core/src/hash/mod.rs

```diff
@@ -900,7 +900,6 @@ mod impls {
 
     macro_rules! maybe_tuple_doc {
         ($a:ident @ #[$meta:meta] $item:item) => {
-            #[cfg_attr(not(bootstrap), doc(fake_variadic))]
```

Comment:
shouldn't this be unconditional, instead of being removed?
```suggestion
            #[doc(fake_variadic)]
```
cc @rust-lang/rustdoc , this should have a test

Reply:
I don't think having tests for each one of these (many) attributes makes sense. This is just an easy mistake to make while we haven't automated this bumping -- it should be pretty easy for the not(bootstrap) case.